src/Umbraco.Web.UI.Client/src/views/components/umb-mini-list-view.html

```diff
@@ -75,13 +75,13 @@ <h4 class="umb-mini-list-view__title-text">{{ miniListView.node.name }}</h4>
                 </div>
 
                 <!-- Load indicator when the list doesn't have items -->
```

Comment:
This comment here is incorrect, as the following code is not about the load indicator, right? We should either remove it, or replace it with the truth.

Reply:
Updated comment, thanks @rbottema.